### PR TITLE
Fix adminrouter config

### PR DESCRIPTION
**Pull request by KostiantynOleksiienko**
## High-level description

The PR adds two files to dcos-config-windows.yaml and change nginx config windows platform.

## Corresponding DC/OS tickets (required)

  - [DCOS_OSS-5834](https://jira.mesosphere.com/browse/DCOS_OSS-5834) JIRA title / short description.

## Related tickets (optional)

  - [DCOS-ID](https://jira.mesosphere.com/browse/DCOS-<number>) JIRA title / short description.

## Checklist for component/package updates:

If you are changing components or packages in DC/OS (e.g. you are bumping the sha or ref of anything underneath `packages`), then in addition to the above please also include:

  - [ ] Change log from the last version integrated (this should be a link to commits for easy verification and review): [example](https://github.com/dcos/dcos-mesos-modules/compare/f6fa27d7c40f4207ba3bb2274e2cfe79b62a395a...6660b90fbbf69a15ef46d0184e36755881d6a5ae)
  - [ ] Included a test which will fail if code is reverted but test is not. If there is no test please explain.
  - [ ] Test Results: [link to CI job test results for component]
  - [ ] Code Coverage (if available): [link to code coverage report]

### CODE REVIEW

**Comment by KostiantynOleksiienko** [comment]
@mesosphere-mergebot request-review @jongiddy

**Comment by KostiantynOleksiienko** [comment]
@mesosphere-mergebot request-review @DominikDary

**Comment by KostiantynOleksiienko** [comment]
@mesosphere-mergebot changelog-not-required

**Comment by KostiantynOleksiienko** [comment]
@mesosphere-mergebot label Work In Progress

File: gen/dcos-config-windows.yaml [hunk 1]
```diff
@@ -212,3 +212,13 @@ package:
...
+  - path: {{ windows_dcos_install_path }}\etc\adminrouter\adminrouter-tls-agent.conf
+    content: |
+      # Note that Agent Admin Router only serves cluster-internal clients. Hence,
+      # browser compatibility is not a criterion for the TLS cipher suite selection.
+      ssl_ciphers EECDH+AES128:RSA+AES128:EECDH+AES256:RSA+AES256:!MD5;
+      ssl_prefer_server_ciphers on;
+      ssl_protocols TLSv1.2;
+  - path: {{ windows_dcos_install_path }}\etc\adminrouter\adminrouter-listen-open.conf
+    content: |
+        listen 61001 default_server;
```
Comment (DominikDary) [inline gen/dcos-config-windows.yaml]: Does this really makes sense in open DC/OS?
/cc @jongiddy

Comment (KostiantynOleksiienko) [reply gen/dcos-config-windows.yaml]: We have the same for the Linux platform and it works for the Windows platform. I have created an open cluster and tested these files on it.

**Comment by jongiddy** [comment]
@mesosphere-mergebot label Ready For Review

**Pull request merged**